.travis.yml

```diff
@@ -1,15 +1,20 @@
 language: objective-c
+env:
+  matrix:
+    - FORMULA_VERSION=
+    - FORMULA_VERSION=--HEAD
 before_install:
   - brew update
   - brew --env
   - brew config
 install:
-  - mkdir -p $(brew --repo)/Library/Taps/neovim
-  - ln -s $PWD $(brew --repo)/Library/Taps/neovim/homebrew-neovim
-  - export PATH="$PWD/bin:${PATH}"
+  - mkdir -p "$(brew --repo)/Library/Taps/neovim"
+  - ln -s "$PWD" "$(brew --repo)/Library/Taps/neovim/homebrew-neovim"
+  - export PATH="$PWD/bin:$PATH"
 script:
-  - brew install neovim --HEAD --verbose
+  - brew install neovim $FORMULA_VERSION --verbose
   - brew head-audit neovim --strict
```

Comment:
Since there is a stable version, I think you can remove the head-audit hack and using `brew audit` directly.